app/models/user.rb

```diff
@@ -20,4 +21,16 @@ def self.search(query)
   def multiple_suppliers?
     suppliers.count > 1
   end
+
+  def create_with_auth0
+    auth0_client = Auth0Api.new.client
+    auth0_response = auth0_client.create_user(
+      name,
+      email: email,
+      email_verified: true,
+      password: SecureRandom.urlsafe_base64,
+      connection: 'Username-Password-Authentication',
+    )
+    update auth_id: auth0_response.fetch('user_id')
```

Comment:
Are there any side effects should Auth0 respond with a non-2xx status here?

Reply:
So I wrote this twice, the first time catching Airbrake errors and showing them to the user, but I decided that was really messy, and thought we should get a Rollbar error if Airbrake does error, rather than showing the user.